vagrant/boxes.d/99-local.yaml.example

```diff
@@ -69,3 +68,7 @@ centos7-dynflow-devel:
     variables:
       dynflow_devel_github_username: '<REPLACE ME>'
       dynflow_devel_github_fork_remote_name: 'origin'
+
+centos7-katello-devel-stable:
+  box_name: centos7-katello-devel-stable
+  hostname: centos7-katello-devel-stable.example.com
```

Comment:
Should users be optionally setting:

```
    variables:
      katello_devel_github_username: <REPLACE ME>
```

The crux of my question is, how do users using the stable box configure their forks automatically?

Reply:
I haven't figured out this part yet, I have to see how that works in forklift and if it can be done after the box is spun up.

Since the whole install is happening automatically and not by a single user, any customization will have to be done after the box (with katello already installed) is spun up, most likely in a playbook.

Reply:
Might be a good reason to extract some capabilities like this to a devel playbook or encouraging the use of shared folders to the vagrant box.